sherpa/utils/__init__.py

```diff
@@ -369,10 +380,6 @@ def requires_pylab(test_function):
 eps = numpy.finfo(numpy.float32).eps
 
 
-def erfinv(y):
```

Comment:
Was this removed because it ends up not being used anywhere in sherpa? It might be a useful function to have for users (but would need documentation and tests and a commitment from us it's worth keeping).